wagtailmetadata/templatetags/wagtailmetadata_tags.py

```diff
@@ -7,8 +7,8 @@
 
 @register.simple_tag(takes_context=True)
 def meta_tags(context, model=None):
-    request = context['request']
+    request = context.get('request', None)
     if not model:
-        model = context['self']
+        model = context.get('self', None)
 
-    return tags.meta_tags(request, model)
+    return tags.meta_tags(request, None)
```

Comment:
@seb-b do you really want to return `None`? I think what you want is returning `model`?

Reply:
good catch! must of left it in while testing

Reply:
Oh that's in an old diff, if you look at the final diff it's the way it should be

https://github.com/neon-jungle/wagtail-metadata/pull/55/files#diff-4c974f3cfb33cd27f25309e10c1c9ad3R14

Reply:
@seb-b oh yes you are right, sorry